### PR TITLE
Add learning group

### PR DESCRIPTION
**Pull request by chobeat**

### CODE REVIEW

File: _events/4.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title:  "Learning Group Session #1"
+date:   2019-08-22 xx:xx:00 +0200
```
Comment (shushugah) [inline _events/4.md]: date:   2019-08-22 19:00:00 +0200

File: learning.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+permalink: /learning
+title: Berlin Tech Worker Coalition Learning Group
```
Comment (shushugah) [inline learning.md]: I'd prefer `Berlin Learning Group`, even in desktop, it doesn't fit the whole screen. See screenshot below 


Screen Shot 2019-07-30 at 19 17 11

**Review by shushugah — CHANGES_REQUESTED** [review]
This looks good. I commented on two small things, namely time and length of title in `/learning` page

File: learning.md (hunk 2 above)
Comment (chobeat) [reply learning.md]: totally right

**Pull request merged**